news/2025/2025-02-27-introducing-teams.md

```diff
@@ -0,0 +1,68 @@
+
+![](/wiki/shared/news/2025-02-27-introducing-teams/team-banner.jpg)
+
+Teams are shown besides usernames in places like user profiles, leaderboards, and forums. Each team gets a flag and name to display everywhere, a unique acronym to go along with it, a banner and description, and their own internal leaderboard.
+
+![](/wiki/shared/news/2025-02-27-introducing-teams/team-display.png)
+
+Users can interact with all of these features by going to the ["My Team"](https://osu.ppy.sh/teams) page at any time. Players not yet in a team will be given the option to create one. Alternatively, by going to the page of an existing team, users will be able to apply to any team they see fit!
+
+### Features
+
+While the feature is still in its infancy, there are plenty of things users can already do with teams:
+
+- Create and join other users' teams.
+- Manage incoming applications.
+- Compete with your peers in the team's leaderboard.
+- Proudly display your allegiance with a team flag and unique acronym!
+
+You can expect to see more features in the future as the system is refined, such as displaying team leaderboards in osu!(lazer) or team versus team matches.
+
+### Limitations
+
+Despite the many features already introduced from the get-go, there are still some limitations to keep in mind:
+
+- Team acronyms are unique.
+- Both team acronyms and names are permanent.
+- Team leaders cannot be swapped.
+- The maximum team size is dependent on the amount of [osu!supporters](/wiki/osu!supporter) on the team.
+  - Each supporter will add 4 slots to the base team size of 8, up to a cap of 256.
+- Currently, the web does not dispose of a listing with all available teams in the platform.
+  - This also applies to the [osu!api (v2)](https://osu.ppy.sh/docs/index.html) which does not have an endpoint for teams.
+
+Some of these limitations may be lifted as the system is iterated upon, a lot of which may depend on the community's feedback. Don't be afraid to propose and discuss changes in the [osu!web GitHub repository](https://github.com/ppy/osu-web/issues?q=is:issue%20state:open%20teams%20label:area:team)!
+
+In the meantime, if team leaders wish to change their team's name, acronym, or even the leader itself, they may do so by contacting the [account support team](/wiki/People/Account_support_team) via [accounts@ppy.sh](mailto:accounts@ppy.sh).
+
+## Stats
+
+It's hard to put into perspective how much the community has grown to love teams ever since their release despite their recency. So why not put it into numbers instead! [MegaMix](https://osu.ppy.sh/users/18152711) has taken the time to compile an [unofficial team list](https://docs.google.com/spreadsheets/d/1z0ddlo3EudoNJdEssrm3tMQYOQY7YQEdXWhMN7JV2TM?rm=minimal).
```

Comment:
Nah. Fuck this shit off.

Reply:
Roger. One of the main reasons I wanted you to check as I was already miffed at the fact these were scraped. Will use the listed stats below, thanks for listing them!